js/script.js

```diff
@@ -0,0 +1,32 @@
+function youtubeIframe() {
+
+	let youtubeBlock = document.querySelectorAll( "#youtube" );
```

Comment:
You use id for searching all divs, but you can use only one id in one page, so how about using class for searching and delete id?